sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_internal/polling.py

```diff
@@ -11,3 +26,30 @@ def __init__(self):
 
     def get_final_get_url(self, pipeline_response):
         return None
+
+
+class KeyVaultBackupClientPollingMethod(LROBasePolling):
+    def get_continuation_token(self):
+        # type() -> str
+        return base64.b64encode(self._operation.get_polling_url().encode()).decode("ascii")
+
+    @classmethod
+    def from_continuation_token(cls, continuation_token, **kwargs):
+        """continuation_token is expected to be the URL of a pending operation"""
+        # type(str, Any) -> Tuple
+        try:
+            client = kwargs["client"]
+        except KeyError:
+            raise ValueError("Need kwarg 'client' to be recreated from continuation_token")
+
+        try:
+            deserialization_callback = kwargs["deserialization_callback"]
+        except KeyError:
+            raise ValueError("Need kwarg 'deserialization_callback' to be recreated from continuation_token")
+
+        continuation_url = base64.b64decode(continuation_token.encode()).decode("ascii")
+        initial_response = client._pipeline.run(client.get(continuation_url), stream=False)
```

Comment:
Okay, doing something like this would return a PipelineResponse from from_continuation_token(), as expected -- so everything works great with very few changes. _But_, the difficulty (as far as I can tell) is doing something similar in the async polling method, when `client` is an async client. How would sending a request from here work in that case?